docs/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## v0.2.0
+
+* GitHub actions now output step summaries: these are visible in workflow
+  run pages on Github (#96)
```

Comment:
Should we make the references real links?

Reply:
I think the Github release (and so the dependabot message) should look fine as is. Maybe you're still right though, will consider next time.

Reply:
Yeah, "naked" links like #96 only works in the UI, in markdown docs they need to be "real" links like `[#96](...issues/96)`